x-pack/plugins/observability_solution/inventory/public/components/grouped_inventory/inventory_summary.tsx

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React from 'react';
+import { EuiFlexGroup, EuiFlexItem, useEuiTheme } from '@elastic/eui';
+import { css } from '@emotion/react';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { GroupSelector } from './group_selector';
+
+export function InventorySummary({
+  totalEntities,
+  totalGroups,
+}: {
+  totalEntities?: number;
+  totalGroups?: number;
+}) {
+  const { euiTheme } = useEuiTheme();
+  const groupCountCss = css`
+    font-weight: ${euiTheme.font.weight.semiBold};
+    border-right: ${euiTheme.border.thin};
+    margin-right: ${euiTheme.size.base};
+    padding-right: ${euiTheme.size.base};
+  `;
+
+  const isGrouped = totalGroups !== undefined;
+
+  return (
+    <EuiFlexGroup gutterSize="none" justifyContent="spaceBetween" alignItems="center">
+      <EuiFlexItem grow={false}>
+        <EuiFlexGroup gutterSize="none">
+          {totalEntities !== undefined && (
+            <EuiFlexItem grow={false}>
+              <span
+                data-test-subj="inventorySummaryEntitiesTotal"
+                css={groupCountCss}
+                style={{ borderRight: !isGrouped ? 'none' : undefined }}
```

Comment:
nit: Is this needed?

Reply:
The little border/vertical separator is part of the designs, so yes, but I'll tidy things a bit more here anyway.

Reply:
This rule sets the borderRight to none or undefined so I don't think it does anything 🤔